docs/builds.rst

```diff
@@ -88,18 +95,66 @@ The artifacts should end up in ``self.version.project.artifact_path(version=self
 Where ``type`` is the name of your builder.
 All files that end up in the artifact directory should be in their final form.
 
-Packages installed in the build environment
--------------------------------------------
-
-The build server does have a select number of C libraries installed, because they are used across a wide array of python projects. We can't install every C library out there, but we try and support the major ones. We currently have the following libraries installed:
-
-    * doxygen
-    * LaTeX (texlive-full)
-    * libevent (libevent-dev)
-    * dvipng
-    * graphviz
-    * libxslt1.1
-    * libxml2-dev
+The build environment
+---------------------
+
+The build process is executed inside Docker containers,
+by default the image used is ``readthedocs/build:2.0``,
+but you can change that using a :doc:`configuration file <yaml-config>`.
+
+Each image is described below.
+
+2.0 (stable)
+~~~~~~~~~~~~
+
+:O.S: Ubuntu 16.04
+:Conda: Miniconda 4.3.31
+:Python:
+  * ``m2crypto``
+  * ``matplolib``
+  * ``numpy``
+  * ``scipy``
+  * ``pandas``
+  * ``pip``
+:Other packages:
+  * ``libxml2-dev``
+  * ``libxslt1.1``
+  * ``libevent``
+  * ``textlive-full``
+  * ``graphviz``
+  * ``pandoc``
+  * ``doxygen``
+  * ``libjpeg``
```

Comment:
Not sure if all these packages are relevant, I keep the most that were in the previous description